abi/sector.go

```diff
@@ -299,6 +300,98 @@ func (p RegisteredSealProof) RegisteredUpdateProof() (RegisteredUpdateProof, err
 	return info.UpdateProof, nil
 }
 
+// mapping to porep_id values
+// https://github.com/filecoin-project/rust-filecoin-proofs-api/blob/9b580c2791f028b7ce3aaef0cf9d68956c50170d/src/registry.rs#L32
+var registeredProofIds = map[RegisteredSealProof]uint64{
+	RegisteredSealProof_StackedDrg2KiBV1:   0,
+	RegisteredSealProof_StackedDrg8MiBV1:   1,
+	RegisteredSealProof_StackedDrg512MiBV1: 2,
+	RegisteredSealProof_StackedDrg32GiBV1:  3,
+	RegisteredSealProof_StackedDrg64GiBV1:  4,
+
+	RegisteredSealProof_StackedDrg2KiBV1_1:   5,
+	RegisteredSealProof_StackedDrg8MiBV1_1:   6,
+	RegisteredSealProof_StackedDrg512MiBV1_1: 7,
+	RegisteredSealProof_StackedDrg32GiBV1_1:  8,
+	RegisteredSealProof_StackedDrg64GiBV1_1:  9,
+}
+
+func (p RegisteredSealProof) porepNonce() uint64 {
+	// https://github.com/filecoin-project/rust-filecoin-proofs-api/blob/9b580c2791f028b7ce3aaef0cf9d68956c50170d/src/registry.rs#L166
+	return 0
+}
+
+// PoRepID produces the porep_id for this RegisteredSealProof. Mainly used for
+// computing replica_id.
+func (p RegisteredSealProof) PoRepID() ([32]byte, error) {
+	// https://github.com/filecoin-project/rust-filecoin-proofs-api/blob/9b580c2791f028b7ce3aaef0cf9d68956c50170d/src/registry.rs#L174
```

Comment:
This line no might not be valid post any change to this file. Maybe add the func name in the comment?

Reply:
This links to a commit, not the current branch